blocks/link-group/link-group.css

```diff
@@ -0,0 +1,92 @@
+.link-group-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-between;
+}
+
+.link-group-wrapper {
+  width: 100%;
+  border-bottom: 0.5px #a9b4be solid;
+}
+
+.link-group-container > div {
+  margin: 0;
+}
+
+.block.link-group ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: none;
+}
+
+.block.link-group ul li {
```

Comment:
Should be replaced with design tokens / variables:
  --udexTypographyBodyXSFontSize: 14px;
  --udexTypographyBodyXSFontFamily: "72 Brand", "Arial", "Helvetica", "sans-serif";
  --udexTypographyBodyXSFontWeight: 400;
  --udexTypographyBodyXSLineHeight: 21px;